README.md

```
@@ -1,67 +1,79 @@
- - v2 (VS 2013 / NuGet 2.x): `https://www.myget.org/F/qwiq/api/v2`
+- v3 (VS 2015+ / NuGet 3.x): `https://www.myget.org/F/qwiq/api/v3/index.json`
+- v2 (VS 2013 / NuGet 2.x): `https://www.myget.org/F/qwiq/api/v2`
 
 Once the feed is configured, install via the nuget UI or via the nuget package manager console
 
 ### Install Core
+
 From the NuGet package manager console
 NuGet
+
 ```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)